webhook_server/tests/test_github_api.py

```
@@ -2394,3 +2394,116 @@ async def test_process_review_thread_skips_when_conversation_resolution_disabled
+                },
+                "auto": {
+                    "created": True,
+                    "source": False,
+                    "revision": True,
+                    "version": False,
+                    "title": True,
+                },
+            },
+        }
+
+        def mock_get_value(value: str, return_on_none: Any = None, extra_dict: Any = None) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `return_on_none`

(ANN401)